javascript/javascript1/week4/voiceassistent.js

```diff
@@ -0,0 +1,104 @@
+"use strict"
+
+//voice Assistant
+let name='';
+let todos=[];
+
+function getReply(command){
+    const commandArray=command.split(' ');
+    
+
+    if(typeof command!=="string"){
+        return "please enter the command in specific formate";
+    }
+    
+    else if(commandArray[0]==="hello"){
```

Comment:
There are string methods like 'startsWith' and 'endsWith' .Informing for your reference
` const commandArray=command.split(' ');

  if (commandArray.startsWith('hello my name is')) {
    name= commandArray[commandArray.length-1];
    return `Nice to meet you ${name}`;
  }`

Reply:
yes,i made the changes